Resources/config/admin-phpcr.xml

```diff
@@ -32,6 +32,10 @@
             <call method="setRouteRoot">
                 <argument>%cmf_routing.dynamic.persistence.phpcr.admin_basepath%</argument>
             </call>
+
+            <call method="setControllerResolver">
+                <argument type="service" id="controller_resolver" />
+            </call>
```

Comment:
i know this is unused in the base class, but as it was protected, extending classes might rely on it.